INSTALLATION.md

```diff
@@ -1,259 +1,250 @@
 
-Talawa API is mainly written and built using `node.js` and `typescript`. We are providing some common approaches to set it up on your system.
+Best way to install and manage node.js is making use of node managers. Two most popular node managers right now are [fnm](https://github.com/Schniz/fnm) and [nvm](https://github.com/nvm-sh/nvm). We'd recommend fnm because it's much faster than nvm. Install whichever one you want and follow their guide to set up node.js on your system.
 
-# Preparation
+# Install git on your system
 
-These steps will help you install API files.
+Follow the setup guide on official [git docs](https://git-scm.com/downloads). Basic git knowledge is required for open source contribution so make sure you're comfortable with it. [Here's](https://youtu.be/apGV9Kg7ics) a good tutorial to get started with git and github.
 
-## Clone the Repository
+# Clone this repository
 
-You will need to download the API files to your system. Here's how.
+First you need a local copy of talawa-api. Run the following command in the directory of choice on your local system.
 
-1.  Clone this repo to your local machine:    
+        git clone https://github.com/PalisadoesFoundation/talawa-api
 
-    ```
-    git clone https://github.com/PalisadoesFoundation/talawa-api
-    ```
+This will download a local copy of talawa-api in that directory.
 
-1.  Change directory to cloned folder:
+# Change directory into the cloned repo(important)
 
-    ````
-    cd talawa-api
-    ````
+**Note:-**` All the commands we're going to execute in the following instructions will assume you are in the root directory of the project. If you fail to do so, the commands will not work.`
 
-## Copy the `.env` Configuration File
+Right after cloning the repo you can change the directory of your current terminal(shell) to the root directory of cloned repository using this command:-
 
-Talawa-API uses a configuration file named `.env` in the root directory. It is not a part of the repo and you will need to create it. There is a sample configuration file named `.env.sample` in the root directory. The `.env.sample` file includes comments for each parameter.
+        cd ./talawa-api
 
+# Creating .env file
 
-1. Create a new `.env` file by copying the contents of the `.env.sample` file.
+A file named `.env` is required in the root directory of talawa-api for storing environment variables required by talawa-api. It is not a part of the repo and you will need to create it. There is a sample file named `.env.sample` in the root directory. Create a new `.env` file by copying the contents of the `.env.sample` file.
 
-    ```
-    cp .env.sample .env
-    ```
+        cp .env.sample .env
 
-1. You will then need to populate the `.env` file with the following paramters. This document will guide you as to how to add the correct values to each.
+This .env file must be populated with the following environment variables for talawa-api to work:-
 
-| Variable             | Description |
```

Comment:
Please restore the table and add some brief descriptions of what each variable does.